dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java

```diff
@@ -91,21 +107,17 @@ private void initialize() {
 						"Error loading annotations file!"));
 		gffLoadService.graphProperty().bind(graphProperty);
 
-		animation = new LeftSlideAnimation(fileOpenPane, WIDTH, ANIM_DURATION, Position.LEFT);
-		bindDisabledFieldsAndButtons();
 	}
 
 	/*
-	 * Disables the openbutton when either no newick file or no node file is selected.
+	 * Disables the openbutton as long as no newick file and no node file is selected.
 	 */
 	private void bindDisabledFieldsAndButtons() {
 		BooleanBinding emptyGraphFile = nodeField.textProperty().isEmpty()
-				.or(edgeField.textProperty().isEmpty());
+				.or(edgeField.textProperty().isEmpty()).or(newickField.textProperty().isEmpty());
 		gffField.disableProperty().bind(graphProperty.isNull().and(emptyGraphFile));
-		// At least both graph files or the newick file must be filled.
-		openButton.disableProperty().bind(emptyGraphFile
-				.and(newickField.textProperty().isEmpty())
-				.and(gffField.disableProperty()));
+		// At least both graph files and the newick file must be filled.
```

Comment:
Because of the change in annotation batchinsert, gff files need to be loaded as well